autoswitch_virtualenv.plugin.zsh

```diff
@@ -26,22 +42,23 @@ function check_venv()
 
         SWITCH_TO=""
 
-        if [[ -f ".venv" ]]; then
-          file_owner="$(stat -c %u .venv)"
-          file_permissions="$(stat -c %a .venv)"
+        venv_path=$(check_venv_path "$PWD")
+        if [[ -n "$venv_path" ]]; then
+          file_owner="$(stat -c %u "$venv_path")"
+          file_permissions="$(stat -c %a "$venv_path")"
 
           if [[ "$file_owner" != "$(id -u)" ]]; then
             echo "AUTOSWITCH WARNING: Virtualenv will not be activated"
             echo ""
             echo "Reason: Found a .venv file but it is not owned by the current user"
-            echo "Change ownership of .venv to '$USER' to fix this"
+            echo "Change ownership of $venv_path to '$USER' to fix this"
           elif [[ "$file_permissions" != "600" ]]; then
             echo "AUTOSWITCH WARNING: Virtualenv will not be activated"
             echo ""
             echo "Reason: Found a .venv file with weak permission settings ($file_permissions)."
             echo "Run the following command to fix this: \"chmod 600 .venv\""
           else
-            SWITCH_TO="$(<.venv)"
+            SWITCH_TO="$(<"$venv_path")"
             AUTOSWITCH_PROJECT="$PWD"
```

Comment:
it might make sense to remove this code if this PR is merged, but I can clear this up myself at a later stage.